### PR TITLE
Login page css updated

### PR DESCRIPTION
**Pull request by nil-kul007**
This is CSS issue fixed.

### CODE REVIEW

File: src/Screens/Login.css [hunk 1]
```diff
@@ -25,6 +25,7 @@
     display: flex;
     flex-direction: column;
     width: 215px;
-    height: 200px;
-    justify-content: center;
+    height: 300px;
```
Comment (archanasn2017) [inline src/Screens/Login.css:28]: Looks 300px is too long can you make it 250px

**Review by archanasn2017 — COMMENTED** [review]
check the comments

File: src/Screens/Login.css (hunk 1 above)
Comment (nil-kul007) [reply src/Screens/Login.css]: Updated to 250 px

**Comment by nil-kul007** [comment]
Resolved

**Pull request merged**